package.json

```diff
@@ -61,8 +61,8 @@
     "node-libs-browser": "^0.5.2",
     "radium": "^0.15.1",
     "raw-loader": "^0.5.1",
-    "react": "^0.14.3",
-    "react-dom": "^0.14.3",
+    "react": "^0.14.1",
+    "react-dom": "^0.14.1",
     "react-matterkit": "0.0.34",
```

Comment:
These are just dev dependencies. They are shouldn't matter if you just install ```react-gsap-enhancer``` with npm.